docs/source/what_is_jupyter.md

```diff
@@ -0,0 +1,412 @@
+  that defines their behavior, and JupyterLab's interface is no exception).
+
+- Jupyter Server (LINK) runs in the background, and it orchestrates the creation,
+  management of, and communication with, your notebook kernels while you're
+  running your notebooks.
+
+- Jupyter kernels themselves are independent programs (operating system
+  processes), and each one is its own REPL in whatever language you
+  requested (commonly Python, though hundreds of other languages and
+  kernels are also available LINK_HERE).
+
+With this design, any program (that you enable) can talk to your kernels
+using common internet communication technologies like HTTP. This gives
+you the power to edit, view, share, and manipulate your data across many
+different programs. One kernel can even connect to multiple editing programs
+simultaneously! (Similarly, in JupyterLab, you can connect a notebook and a
+console to the same kernel!)
+
+Project Jupyter actually defines a standard (LINK) that other programs can
+follow that will allow them to hook into your kernels in virtually any way
+you can imagine. You can invent new editing and viewing experiences for your
+data this way, using the interactive computing capabilities provided by
+the kernels.
+
+## Benefits of a many-piece design
+
+By breaking up a program like JupyterLab into multiple component pieces, you
+can customize the software to meet your needs. If one piece is missing something,
+you can replace it with a custom version made by yourself or another person to
+add whatever features you would like to see.
+
+You can also invent completely new experiences using those pieces that the
+designers may not have imagined when they started, and they can often inter-
+operate seamlessly with existing Jupyter software.
+
+Because Jupyter Server provides kernel communication and management features
+in a cohesive, self-contained package, for instance, new notebook editor
+programs can focus solely on adding new interface and editing experiences,
+leaving the task of creating and managing new kernels completely up to the
+Jupyter Server.
+
+JupyterCAD is a 3D modeling tool built on top of Jupyter software, for
+instance, and there are other examples too (LINK HERE).
+
+## Talking to Kernels (The Jupyter Protocol)
+
+Anyone can make new software that talks to Jupyter's kernels (read more
+about those above), by using the Jupyter Protocol. The protocol provides
+a standardized blueprint for passing information back and forth between
+the kernels and other software.
+
+By implementing the designs described in the Jupyter Protocol, you could
+invent a completely new interactive programming experience, or add support
+for a new programming language to Jupyter.
+
+Because Project Jupyter is free and open, it encourages anyone to explore
+new ways of working with their notebooks and kernels, and likes to offer
+compatibility and interoperation with other software.
+
+TODO add more here? examples? etc.
+
```

Comment:
I'm concerned that this section is getting into significant technical detail that is not necessary for people just getting started with Jupyter. I feel like this document should be "100-level" (in US university terms), but this block is 200-level information, for people who want to know more.